Missing dependency json5 in npm package

## Expected Behavior

I have a jscad cli project. When I did `npm upgrade` I was no long able to compile my models with jscad.

## Actual Behavior

After upgrading to the latest version, when I run jscad on the command line I get an error about missing json5 dependency. It appears it was added to jscad [8 days ago](https://github.com/jscad/OpenJSCAD.org/commit/a33a66151ef79d62443333275e6de72fe18d0921), but was not added to package.json for @jscad/core.

## Steps to Reproduce the Problem

  1. `npm i @jscad/cli`
  2. `npx jscad autopilot.js`

Error:
```
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module 'json5'
Require stack:
- /example/node_modules/@jscad/core/src/parameters/getParameterDefinitionsFromSource.js
- /example/node_modules/@jscad/core/src/code-loading/webRequire.js
- /example/node_modules/@jscad/core/src/code-loading/loadDesign.js
- /example/node_modules/@jscad/core/src/code-evaluation/rebuildGeometry.js
- /example/node_modules/@jscad/core/src/code-evaluation/index.js
- /example/node_modules/@jscad/core/src/index.js
- /example/node_modules/@jscad/cli/src/generateOutputData.js
- /example/node_modules/@jscad/cli/cli.js
```

## Specifications

  - Version: @jscad/core 2.3.7
  - Platform: Linux CLI
  - Environment: (browser, local server, node.js etc) node.js

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -39,6 +39,7 @@
     "@jscad/io-utils": "2.0.7",
     "@jscad/modeling": "2.5.2",
     "@jscad/vtree": "2.0.8",
+    "json5": "^2.2.0",
     "strip-bom": "4.0.0"
   },
   "devDependencies": {

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -39,7 +39,7 @@
     "@jscad/io-utils": "2.0.7",
     "@jscad/modeling": "2.5.2",
     "@jscad/vtree": "2.0.8",
-    "json5": "^2.2.0",
+    "json5": "2.2.0",
     "strip-bom": "4.0.0"
   },
   "devDependencies": {

--- a/packages/web/package.json
+++ b/packages/web/package.json
@@ -47,7 +47,6 @@
     "es2015-i18n-tag": "1.6.1",
     "file-saver": "2.0.2",
     "is-electron": "2.2.0",
-    "json5": "^2.2.0",
     "morphdom": "2.6.1",
     "most": "1.8.0",
     "most-proxy": "3.3.0",

